python-checks/src/test/java/org/sonar/python/checks/quickfix/PythonQuickFixVerifier.java

```diff
@@ -115,7 +117,7 @@ private static List<PreciseIssue> getIssuesWithQuickFix(PythonCheck check, Strin
 
     PythonVisitorContext visitorContext = new PythonVisitorContext(parse,
       pythonFile, null, "",
-      ProjectLevelSymbolTable.empty());
+      ProjectLevelSymbolTable.empty(), null);
```

Comment:
If you have a dummy implementation for `CacheContext` in the `python-checks-testkit`, I believe you will be able to use it here as well, which would also avoid to pass `null` here.

edit: In one of my next comments, I suggest to have such an implementation in the `frontend` instead, which could be reused here.